lib/folio_client.rb

```diff
@@ -120,7 +123,8 @@ def put(path, body = nil, content_type: "application/json")
   def connection
     @connection ||= Faraday.new(
       url: config.url,
-      headers: DEFAULT_HEADERS.merge(config.okapi_headers || {})
+      headers: DEFAULT_HEADERS.merge(config.okapi_headers || {}),
+      request: {timeout: 120}
```

Comment:
I also tried setting this to `60`, but `120` was where it made a difference.

Reply:
Shelley and I looked at logs and it's possible kafka is continuing to be the bottleneck.